Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm

```diff
@@ -335,5 +336,13 @@ static void disableDowngradeToLayoutManager()
     return isSystemWebKit;
 }
 
+void AuxiliaryProcess::setNotifyOptions()
+{
+#if ENABLE(NOTIFY_BLOCKING)
+    notify_set_options(NOTIFY_OPT_DISPATCH);
```

Comment:
The commit message says "We only
need to set notify options the original way on macOS versions that do not have the feature
ENABLE(NOTIFY_BLOCKING)." but here we set the options when ENABLE(NOTIFY_BLOCKING). Is this intended change?

Reply:
Ah, good catch! The change is intended, since we always would like to set the dispatch option. I will fix the commit message.